### PR TITLE
Fixed profile name being pushed under section

### PR DESCRIPTION
**Pull request by Ralphvandodewaard**
For issue: https://bitbucket.org/kickorstick/anchor-chat-client/issues/501/profile-name-not-visible-on-about-me-and

### CODE REVIEW

**Comment by Ralphvandodewaard** [comment]
On the About Me page, the username is displayed in a <h1> element, which is followed up by a section containing the Tabs. For a reason I am not sure of, the h1 element would get pushed under this section on screens with lower heights. By putting the h1 element in its own section element, this issue is fixed.

**Comment by IanCStewart** [comment]
This doesn't seem like the most elegant fix. Could you check if there is a better fix?

**Comment by sjaakluthart** [comment]
@IanCStewart will you look for a better fix or will @Ralphvandodewaard finish it?

File: src/profile/styles.js [hunk 1]
```diff
@@ -68,6 +69,7 @@ export default {
     overflow: 'hidden',
     textOverflow: 'ellipsis',
     width: '100%',
-    textAlign: 'center'
+    textAlign: 'center',
+    display: 'table'
```
Comment (sjaakluthart) [inline src/profile/styles.js]: This does not seem like a correct solution, please find another.

**Comment by Ralphvandodewaard** [comment]
Explanation on changes:

- `overflow: hidden` on `header` and `secondaryText` was causing these texts to 'disappear' when  the height of the screen got small enough to start scrolling on the page. By removing the `overflow: hidden`, this issue is resolved.

- The downside of this solution is that long texts on `header` and `secondaryText` will not have the `textOverflow: ellipsis` effect anymore. The overflow of these texts will still be hidden though because of the `overflow: hidden` in the `root`, which is unchanged.

**Comment by Ralphvandodewaard** [comment]
Text that is too long will not overflow, it will still be hidden. This is because of the `overflow: hidden` in `root`. The only visible effect of these changes is that the `textOverflow: ellipsis` is removed now.
examples

**Comment by sjaakluthart** [comment]
I get that long text will not be shown if the root element has `overflow: hidden`... Your commit resolves the symptom, not the problem. Figure out why this issue happens and try to fix that.

**Pull request merged**